browser/net/brave_common_static_redirect_network_delegate_helper.cc

```diff
@@ -70,7 +72,8 @@ int OnBeforeURLRequest_CommonStaticRedirectWorkForGURL(
     replacements.SetQueryStr(request_url.query_piece());
     const base::CommandLine& command_line =
         *base::CommandLine::ForCurrentProcess();
-    if (!command_line.HasSwitch(switches::kUseGoUpdateDev)) {
+    if (!command_line.HasSwitch(switches::kUseGoUpdateDev) &&
+        !base::FeatureList::IsEnabled(features::kUseDevUpdaterUrl)) {
```

Comment:
actually I don't think we should use `base::Feature` for such things, especially here - now there is a mix of a command-line switch and a Feature. Features and `chrome://flags` are mostly for what makes sense to end users / web devs. Can we switch it to a simple command-line flag?

Reply:
No, we need something that can be set by  QA on android.

Reply:
this is critical for QA right now because there is no campaign running in release

Reply:
we will most likely replace this with a different solution later, but we need this ASAP and this is the quickest way to enable it for QA